frr-tools/metrics/collector/bgp_test.go

```diff
@@ -15,46 +15,47 @@ var (
 	metricsTmpl = `
 	# HELP metallb_bgp_announced_prefixes_total Number of prefixes currently being advertised on the BGP session
 	# TYPE metallb_bgp_announced_prefixes_total gauge
-	metallb_bgp_announced_prefixes_total{peer="{{ .NeighborIP }}"} {{ .AnnouncedPrefixes }}
+	metallb_bgp_announced_prefixes_total{peer="{{ .NeighborIP }}", vrf="{{ .NeighborVRF }}"} {{ .AnnouncedPrefixes }}
```

Comment:
(reminder to document this vrf label)

Reply:
Filed https://github.com/metallb/metallb/issues/1751